litmus-portal/frontend/package.json

```diff
@@ -1,122 +1,123 @@
-    "test:coverage": "npm test --coverage --colors --maxWorkers=2"
-  },
-  "dependencies": {
-    "@apollo/client": "^3.1.2",
-    "@date-io/date-fns": "^1.3.13",
-    "@material-ui/core": "^4.10.2",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.57",
-    "@material-ui/pickers": "^3.2.10",
-    "@testing-library/jest-dom": "^5.11.10",
-    "@testing-library/react": "^11.2.6",
-    "ace-builds": "^1.4.12",
-    "brace": "^0.11.1",
-    "clsx": "^1.1.1",
-    "cronstrue": "^1.100.0",
-    "d3": "^5.16.0",
-    "dagre-d3": "^0.6.4",
-    "date-fns": "^2.14.0",
-    "graphql": "^15.3.0",
-    "html2canvas": "^1.0.0-rc.7",
-    "i18next": "^19.7.0",
-    "i18next-browser-languagedetector": "^6.0.1",
-    "i18next-http-backend": "^1.0.19",
-    "js-yaml": "^3.14.0",
-    "jsonwebtoken": "^8.5.1",
-    "jspdf": "^2.1.1",
-    "jspdf-autotable": "^3.5.13",
-    "litmus-ui": "^1.1.0",
-    "localforage": "^1.7.3",
-    "lodash": "^4.17.20",
-    "moment": "^2.27.0",
-    "plotly.js": "^1.58.4",
-    "prop-types": "^15.7.2",
-    "rc-progress": "^3.0.0",
-    "react": "^16.9.0",
-    "react-ace": "^9.2.1",
-    "react-date-range": "^1.0.3",
-    "react-dom": "^16.9.0",
-    "react-google-charts": "^3.0.15",
-    "react-i18next": "^11.7.2",
-    "react-plotly.js": "^2.4.0",
-    "react-redux": "^7.1.3",
-    "react-router-dom": "^5.1.2",
-    "react-scripts": "^3.4.3",
-    "react-simple-maps": "^2.1.2",
-    "redux": "^4.0.1",
-    "redux-devtools-extension": "^2.13.8",
-    "redux-persist": "^6.0.0",
-    "redux-thunk": "^2.3.0",
-    "subscriptions-transport-ws": "^0.9.17",
-    "uuid": "^8.3.1",
-    "yaml": "^1.10.0"
-  },
-  "lint-staged": {
-    "src/**/*.+(js|jsx|ts|tsx)": [
-      "npm run format",
-      "git add"
+    "name": "litmus-portal",
+    "version": "0.1.0",
+    "homepage": "https://litmus-portal.com",
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "cross-env NODE_PATH=src react-scripts build",
+        "eject": "react-scripts eject",
+        "lint": "eslint . --ext .js,.tsx,.ts",
+        "format": "prettier --write \"src/**/*.+(ts|tsx|js|jsx|json)\" && eslint --fix . --ext .js,.tsx,.ts",
+        "test": "react-scripts test --maxWorkers=2",
+        "test:clean": "react-scripts test --clearCache",
+        "test:watch": "npm test --watch",
+        "test:coverage": "npm test --coverage --colors --maxWorkers=2"
+    },
+    "dependencies": {
+        "@apollo/client": "^3.1.2",
+        "@date-io/date-fns": "^1.3.13",
+        "@material-ui/core": "^4.10.2",
+        "@material-ui/icons": "^4.9.1",
+        "@material-ui/lab": "^4.0.0-alpha.57",
+        "@material-ui/pickers": "^3.2.10",
+        "@testing-library/jest-dom": "^5.11.10",
+        "@testing-library/react": "^11.2.6",
+        "ace-builds": "^1.4.12",
+        "brace": "^0.11.1",
+        "clsx": "^1.1.1",
+        "cron-parser": "^3.3.0",
+        "cronstrue": "^1.100.0",
+        "d3": "^5.16.0",
```

Comment:
how did package.json change?

Reply:
I had one extension which was linting it. I have removed it and fixed it.Now it is changed only because i have added one package "cron-parser"